lms/static/sass/views/_login-register.scss

```diff
@@ -447,6 +442,15 @@ $sm-btn-linkedin: #0077b5;
 
     }
 
+    .button-secondary-login {
+        @extend .nav-btn;
+        font-size: 0.8em;
```

Comment:
Please refrain from using ems for units of measurements. Pixels, rems, and pulling from our [typography extends](https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Felements%2F_typography.scss) are your best bet in the current platform.